test/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "Game-test.js",
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha"
```

Comment:
We'll learn more about NPM next week, but for linting purposes, you could add another script here that says:

`"lint": "eslint ./"`

and then you'll be able to run `npm run lint` in your terminal (just like you run your tests) and see the linting output/errors that need to be fixed.